docs/plugins/building_a_plugin/_layer_data_guide.md

```diff
@@ -56,6 +56,11 @@ class ArrayLike(Protocol):
 # typing than literal array types (e.g. numpy, dask, xarray, etc...)
 ```
 
+Note that when writing your own plugin, such type annotations are nearly always optional, 
+except in rare cases like `magicgui` function widgets.
+Here, for clarity, we've provided aliases that indicate the how values of the type are 
```

Comment:
Something is off with this line, can you take another look?

Reply:
yes sorry 😅 , ﻿﻿﻿ec41dda66f3c8a55f647dbe5c07f852e26904cb2